frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -1,124 +1,65 @@
 <template>
-  <card-content-node v-bind="$props">
-    <v-container fluid>
-      <v-row no-gutters class="text-subtitle-2">
-        <v-col cols="2">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column1') }}
-        </v-col>
-        <v-col cols="7">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column2Html') }}
-        </v-col>
-        <v-col cols="2">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column3') }}
-        </v-col>
-        <v-col cols="1" />
-      </v-row>
-
-      <api-sortable
```

Comment:
ApiSortable is not used anywhere else. Couldn't we just adjust ApiSortable with whatever functionality was missing, instead of duplicating to StoryboardSortable?